bucket/foobar2000.json

```diff
@@ -28,22 +29,28 @@
             "    New-Item \"$persist_dir\\playlists\" -ItemType Directory -Force | Out-Null",
             "    if (Test-Path \"$env:Appdata\\foobar2000\") {",
             "        Write-Host -F yellow \"Copying old '$env:Appdata\\foobar2000' to '$persist_dir'\"",
-            "        Copy-Item \"$env:Appdata\\foobar2000\\*\" \"$dir\" -Recurse -Force",
+            "        Copy-Item \"$env:Appdata\\foobar2000\\*\" \"$dir\" -Force -Recurse",
+            "        if (Test-Path \"$env:Appdata\\foobar2000\\playlists-v*\") {",
+            "            Copy-Item \"$dir\\playlists-v*\" \"$persist_dir\\playlists\" -Force -Recurse",
+            "        }",
+            "        if (Test-Path \"$env:Appdata\\foobar2000\\user-components\") {",
```

Comment:
I do not understand this check. 

If there is this folder in appdata, you are copying, user component folder from dir to persist.

Why is there a user-components directory inside appdata, which is not used then. And you are not copying anything from it...

Reply:
It's a migration from when foobar2000 in scoop used to be run in non-portable mode. The line right above there moved the files to `$dir` from AppData already.

Reply:
I'm not the one who wrote that migration code originally, but I'll look into a way to get it to only run when needed as well (currently it will always run if you've _ever_ had foobar2000 installed non-portably).

Reply:
Oh, I see what you mean, I'll fix it.